<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Resize</class>
 <widget class="QDialog" name="Resize">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="titreLayout">
     <item>
      <widget class="QLabel" name="_tLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Taille du rognage</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="xyLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="x0Layout">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <widget class="QLabel" name="_x0Label">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>x0 :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="_x0Input">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="y0Layout">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <widget class="QLabel" name="_y0Label">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>y0 :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="_y0Input">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="xfLayout">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <widget class="QLabel" name="_xfLabel">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>xf :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="_xfInput">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="yfLayout">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <widget class="QLabel" name="_yfLabel">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>yf :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="_yfInput">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="hlLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="lLayout">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <widget class="QLabel" name="_lLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>55</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>55</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>Largeur :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="_lInput">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hLayout">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <widget class="QLabel" name="_hLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>55</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>55</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>Hauteur :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="_hInput">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="_buttonbox">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
       <property name="centerButtons">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>_x0Input</tabstop>
  <tabstop>_y0Input</tabstop>
  <tabstop>_yfInput</tabstop>
  <tabstop>_hInput</tabstop>
  <tabstop>_xfInput</tabstop>
  <tabstop>_lInput</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>_buttonbox</sender>
   <signal>accepted()</signal>
   <receiver>Resize</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_buttonbox</sender>
   <signal>rejected()</signal>
   <receiver>Resize</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
